### PR TITLE
Check security_scan flag

### LINKED ISSUE #1703: Create proper error message ins case of disabled security scan.
**Describe the bug**
In case of disabled security scan would better to create proper error message when  get policies or initiate image scan.

**Steps to reproduce the issue:**
Get policies on cluster where security scan isn't enabled.

**Expected behavior**
Create a clear error message in this case ("security scan isn't enabled")

### PR DESCRIPTION
**Pull request by pbalogh-sa**
| Q               | A
| --------------- | ---
| Bug fix?        | yes
| New feature?    | no
| API breaks?     | no
| Deprecations?   | no
| Related tickets | fixes #1703 
| License         | Apache 2.0

### What's in this PR?

Checking security_scan flag before anchore requests

### Why?

It makes error messages more exact.

### Checklist

- [x] Implementation tested (with at least one cloud provider)
- [x] Error handling code meets the [guideline](https://github.com/banzaicloud/pipeline/blob/master/docs/error-handling-guide.md)
- [ ] Logging code meets the guideline (TODO)
- [ ] OpenAPI and Postman files updated, client regenerated (`make generate-client`) (if needed)
- [ ] User guide and development docs updated (if needed)
- [ ] Related Helm chart(s) updated (if needed)

### To Do

- [ ] If the PR is not complete but you want to discuss the approach, list what remains to be done here

### CODE REVIEW

File: api/imagescan.go [hunk 1]
```diff
@@ -73,6 +73,16 @@ func ScanImages(c *gin.Context) {
 	if !ok {
 		return
 	}
+
+	if !commonCluster.GetSecurityScan() {
+		c.JSON(http.StatusNotFound, pkgCommmon.ErrorResponse{
```
Comment (stoader) [inline api/imagescan.go]: Can't `common.ErrorResponseWithStatus` be used here to send back error response?

Comment (pbalogh-sa) [reply api/imagescan.go]: Thanks for mentioning common.ErrorResponseWithStatus function.

Comment (matyix) [reply api/imagescan.go]: thx @pbalogh-sa

File: api/imagescan.go [hunk 2]
```diff
@@ -18,6 +18,7 @@ import (
 	"net/http"
 	"path"
 
+	"github.com/banzaicloud/pipeline/api/ark/common"
```
Comment (tarokkk) [inline api/imagescan.go]: Is there any specific reason that we import this from api/ark/common and not refactor it to api/common?

Comment (stoader) [reply api/imagescan.go]: Agree this should go into common.

Comment (pbalogh-sa) [reply api/imagescan.go]: I'm on it

**Pull request merged**